README.md

```diff
@@ -1 +1,25 @@
-# high-availability
+# Highly-available 3scale
```

Comment:
This is nice.

Maybe it would be a good idea to put this in common with docs team to identify possible gaps at Product doc level?

Reply:
That's the idea. Products docs can use this repo as a starting point.

Reply:
:+1: